sortElectrodes output file will now be named according to channel

MorgNMont opened the pull request.

Commit:
MM Testing commit

Commit:
MM Testing Commit

dorahermes commented:
@hharveygit - Morgan updated the sortElectrodes file to have the channels path added - it works nicely for me, you can merge if you are ok as well.

MorgNMont committed:
small comment to clarify

hharveygit reviewed (commented):
The updated code currently would not work if channels were given as a table rather than the path to the file.
Solution:
- rework the if/else blocks between line 31 and 39 so that whether the file is saved is tied to the channels input, not electrodes.
- Move line 85 that you added to inside the "if saveFile" block

Other suggestion:
- variable "chann" as it is named right now is not well named, because what you mean by that variable is actually the channel_path. I suggest renaming chann as channel_path and renaming your current channel_path as something like "path_frag"

MorgNMont committed:
Update sortElectrodes.m

In line 29-39, the file that is saved is tied to the channels input, not the electrodes. Switched the positions of channels and electrodes within this loop.

Changes in the following lines addresses the other suggestions @hharveygit suggested.
